operator/EXTENDING_CRDS.md

```diff
@@ -122,3 +122,19 @@ Some data types are discouraged, in particular:
 * Nullability: See [Kubernetes Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting-and-nullable).
 * Late initialization: See [API Conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#late-initialization).
 * Labels, Selector and Annotations: See [API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions).
+
+## Crd-schema-checker
+
+Crd-schema-checker is a tool for finding breaking changes and violations of best practices in CRDs. It can be found [here](https://github.com/openshift/crd-schema-checker)
```

Comment:
```suggestion
Crd-schema-checker is a tool for finding breaking changes and violations of best practices in CRDs. It can be found [here](https://github.com/openshift/crd-schema-checker).
```

Reply:
I moved the link and removed the last sentence so this suggested change became obsolete.